tool.go

```diff
@@ -750,7 +750,17 @@ func runNode(script string, args []string, dir string, quiet bool) error {
 	}
 
 	if runtime.GOOS != "windows" {
-		allArgs = append(allArgs, "--stack_size=10000", script)
+		r := syscall.Rlimit{}
+
+		err := syscall.Getrlimit(syscall.RLIMIT_STACK, &r)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "gopherjs: failed to get stack size limit: %v", err)
+			os.Exit(1)
```

Comment:
What would happen if you did `return err` instead of handling the error right here?